### PR TITLE
fixed user audio update on admin change

### PR DESCRIPTION
**Pull request by alexey-petrov**
Closes:

- https://github.com/sparkletown/internal-sparkle-issues/issues/1214

Added `useEffect` that changes client-side reactions audio status based on the venue value updated at the admin while having 2 pages open simultaneously

### CODE REVIEW

File: src/components/templates/Auditorium/components/Section/Section.tsx [hunk 1]
```diff
@@ -33,9 +33,20 @@ export interface SectionProps {
...
+    hide: disableUserAudio,
+    show: enableUserAudio,
+  } = useShowHide(venue.isReactionsMuted ?? false);
+
+  useEffect(() => {
+    if (!venue.isReactionsMuted) {
+      disableUserAudio();
+    } else {
+      enableUserAudio();
+    }
```
Comment (goran-peoski-work) [inline src/components/templates/Auditorium/components/Section/Section.tsx:48]: Can swap the if and else parts to remove the negation in the if

File: src/components/templates/Auditorium/components/Section/Section.tsx [hunk 2]
```diff
@@ -33,9 +33,20 @@ export interface SectionProps {
...
 
-  const { isShown: isUserAudioOn, toggle: toggleUserAudio } = useShowHide(
-    venue.isReactionsMuted ?? false
-  );
+  const {
+    isShown: isUserAudioOn,
+    toggle: toggleUserAudio,
+    hide: disableUserAudio,
+    show: enableUserAudio,
+  } = useShowHide(venue.isReactionsMuted ?? false);
```
Comment (goran-peoski-work) [inline src/components/templates/Auditorium/components/Section/Section.tsx:41]: Shouldn't there already be a `DEFAULT_` constant somewhere? There is a `worldDefaultSettings.ts` to find these faster for the world. Maybe something similar should exist for the rest.

**Review by goran-peoski-work — APPROVED** [review]
LGTM

File: src/components/templates/Auditorium/components/Section/Section.tsx (hunk 2 above)
Comment (alexey-petrov) [reply src/components/templates/Auditorium/components/Section/Section.tsx]: added const + imported to use here e291c35

File: src/components/templates/Auditorium/components/Section/Section.tsx (hunk 1 above)
Comment (alexey-petrov) [reply src/components/templates/Auditorium/components/Section/Section.tsx]: done e291c35

**Pull request merged**